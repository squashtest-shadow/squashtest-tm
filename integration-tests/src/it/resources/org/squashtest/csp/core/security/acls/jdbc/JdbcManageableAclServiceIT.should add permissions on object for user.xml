<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE dataset SYSTEM "../../../../../../../dtd/squash-tm.1.9.10.dtd">

<dataset>
	<CORE_PARTY PARTY_ID="-10"/>
	<CORE_USER PARTY_ID="-10" LOGIN="batman" LAST_NAME="batman" CREATED_BY="dbu" CREATED_ON="2011-03-19" />
	
	<core_group />
	<core_group_authority />
	<core_group_member />
	
	<ACL_CLASS ID="-1000" CLASSNAME="batmobile" />
	<ACL_OBJECT_IDENTITY ID="-1000" IDENTITY="-1000" CLASS_ID="-1000" />

	<ACL_GROUP ID="-100" QUALIFIED_NAME="driver" />

	<acl_group_permission />
	<acl_responsibility_scope_entry />

</dataset>