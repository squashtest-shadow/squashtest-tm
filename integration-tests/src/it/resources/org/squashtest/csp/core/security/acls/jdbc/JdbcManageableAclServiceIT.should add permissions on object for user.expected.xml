<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE dataset SYSTEM "../../../../../../../dtd/squash-tm.1.9.0.result.dtd">
<dataset>
	<CORE_PARTY PARTY_ID="10" />
	<CORE_USER PARTY_ID="10" LOGIN="batman" LAST_NAME="batman" CREATED_BY="dbu" CREATED_ON="2011-03-19" />
	
	<ACL_CLASS ID="1000" CLASSNAME="batmobile" />
	<ACL_OBJECT_IDENTITY ID="1000" IDENTITY="1000" CLASS_ID="1000" />

	<ACL_GROUP ID="100" QUALIFIED_NAME="driver" />

	<ACL_RESPONSIBILITY_SCOPE_ENTRY PARTY_ID="10" ACL_GROUP_ID="100" OBJECT_IDENTITY_ID="1000" />

</dataset>